flowsa/methods/flowbysectormethods/CAP_HAP_national_2017.yaml

```diff
@@ -28,6 +22,28 @@ _allocation_types:
     helper_from_scale: national
     clean_helper_fba: clean_bls_qcew_fba
     clean_helper_fba_wsec: bls_clean_allocation_fba_w_sec
+  _use_allocation: &use_allocation
```

Comment:
can you add a blank line above this use_allocation for visual effect?

also, I like how `clean_parameter` is shown but commented out, the same as `allocation_flow` above for EIA_MECS. Maybe you could add the `names` parameter as well to both, as a comment, as a flag that it needs to be included for an individual `activity_set`

Reply:
Good ideas!